frontend/src/views/activity/Activity.vue

```diff
@@ -140,10 +140,15 @@ export default {
   },
   computed: {
     availableCampCollaborations () {
-      return this.campCollaborations.map(cc => {
+      const currentCampCollaborationIds = this.activity.campCollaborations().items.map(cc => cc.id)
+      return this.campCollaborations.filter(cc => {
+        return (cc.status === 'established') || (currentCampCollaborationIds.includes(cc.id))
+      }).map(value => {
+        const leaved = value.status !== 'established'
+        const text = value.user().username + (leaved ? ' (Lager verlassen)' : '')
```

Comment:
Hilf mir da kurz: Du schreibst "Lager verlassen" immer hin wenn der Status nicht 'established' ist, also bei 'left' aber auch bei 'invited'. Das macht aber nichts, weil jemand der das erste Mal 'invited' ist gar nirgends in einer Activity drin stehen kann und ergo deshalb hier gar nicht auftaucht. Korrekt?

Reply:
habe die Logik gedreht - ist wohl für die Zukunft stabiler.

Reply:
Stabiler aber ich glaubs nicht unbedingt richtiger :-)
Wenn jemand das Lager verlässt "left" und danach wieder dazu eingeladen wird ist der Status auf "invited". Dennoch wirst du wahrscheinlich im Text noch "Lager verlassen" anzeigen, weil du die Person zu diesem Zeitpunkt noch nicht auswählen kannst.

Mein Kommentar war eher gedacht in der Richtigung: Ich glaube der Code stimmt ist aber nicht auf den ersten Blick intuitiv wieso er stimmt.